components/raftstore/src/coprocessor/dispatcher.rs

```diff
@@ -506,7 +511,7 @@ impl<E: KvEngine> CoprocessorHost<E> {
         }
         for batch in &cmd_batches {
             for cmd in &batch.cmds {
-                self.post_apply(&batch.region, cmd);
+                self.post_apply(&EMPTY_REGION, cmd);
```

Comment:
/cc @NingLin-P PTAL

Reply:
The region here is actually used by the `loop_ob` macro to construct `ObserverContext`, and currently the only observer of `post_apply` is `LockObserver`, which simply ignored `ObserverContext`. But using an empty region here may be unexpected by incoming observers.

Reply:
If it's not used at all, can it be removed completely?